INFO     planner time limit: 900s
INFO     planner memory limit: 3584 MB

INFO     Running translator.
INFO     translator stdin: None
INFO     translator time limit: 899s
INFO     translator memory limit: 3584 MB
INFO     translator command line string: /usr/local/downward-projects/downward/experiments/.venv/bin/python3 /usr/local/downward-projects/downward/experiments/src/thesis/data/thesis-thesis_exp/code-f7a4f68fd467cf7cff4d3dfcfdfcfa2023e44324_e5e41175/builds/release/bin/translate/translate.py domain.pddl problem.pddl --sas-file output.sas
Parsing...
Parsing: [0.010s CPU, 0.006s wall-clock]
Normalizing task... [0.000s CPU, 0.000s wall-clock]
Instantiating...
Generating Datalog program... [0.000s CPU, 0.004s wall-clock]
Normalizing Datalog program...
Normalizing Datalog program: [0.000s CPU, 0.001s wall-clock]
Preparing model... [0.020s CPU, 0.006s wall-clock]
Generated 26 rules.
Computing model... [61.390s CPU, 61.900s wall-clock]
4185514 relevant atoms
2733812 auxiliary atoms
6919326 final queue length
15209866 total queue pushes
Completing instantiation... 
Translator ran out of memory, traceback:
===============================================================================
MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/downward-projects/downward/experiments/src/thesis/data/thesis-thesis_exp/code-f7a4f68fd467cf7cff4d3dfcfdfcfa2023e44324_e5e41175/builds/release/bin/translate/instantiate.py", line 119, in explore
    return instantiate(task, model)
  File "/usr/local/downward-projects/downward/experiments/src/thesis/data/thesis-thesis_exp/code-f7a4f68fd467cf7cff4d3dfcfdfcfa2023e44324_e5e41175/builds/release/bin/translate/instantiate.py", line 92, in instantiate
    inst_action = action.instantiate(
MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError

During handling of the above exception, another exception occurred:

MemoryError
===============================================================================
translate exit code: 20

Driver aborting after translate
INFO     Planner time: 119.10s
search time limit missing -> can't compute time scores
search memory limit missing -> can't compute memory score
